pkgs/by-name/de/deskreen/package.nix

```
@@ -0,0 +1,83 @@
+{ lib
+, fetchurl
+, stdenv
+, appimageTools
+, alsa-lib
+, at-spi2-atk
+, cairo
+, cups
+, dbus
+, expat
+, glib
+, glibc
+, gtk3
+, hicolor-icon-theme
+, libdrm
+, libxkbcommon
+, mesa
+, nspr
+, nss
+, pango
+, xorg
+}:
+
+stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

You can safely use `stdenvNoCC` since you don't need to compile anything in here.